lib/upperkut/batch_execution.rb

```diff
@@ -11,31 +11,26 @@ def initialize(worker, logger = Upperkut::Logging.logger)
 
     def execute
       worker_instance = @worker.new
-      items = @worker.fetch_items.freeze
-
-      items_body = items.collect do |item|
-        item['body']
-      end
+      items = @worker.fetch_items
```

Comment:
we need to freeze here because server_middleware cannot mutate items object, only read.